test/unit/metrics/test_metricframe_comprehensive.py

```diff
@@ -5,6 +5,7 @@
 import pandas as pd
 import pytest
 import sklearn.metrics as skm
+import time
```

Comment:
I think you can get rid of the things in here. Those look like leftovers from when I was benchmarking the performance improvements.